### PR TITLE
{chem}[intel/2016b] DL_POLY_Classic v1.9 w/ PLUMED 2.2.3 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
requires https://github.com/hpcugent/easybuild-easyblocks/pull/829

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2307.phanpy.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ef0606296833304b34a936fbaa55a157 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node1273.muk.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7f61b156a5b616aa27ab2579bd377619 for a full test report.

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/easyconfigs/d/DL_POLY_Classic/DL_POLY_Classic-1.9-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name = 'DL_POLY_Classic'
```
Comment (wpoely86) [inline easybuild/easyconfigs/d/DL_POLY_Classic/DL_POLY_Classic-1.9-intel-2016b.eb]: is there any request for this without plumed?

Comment (boegel) [inline easybuild/easyconfigs/d/DL_POLY_Classic/DL_POLY_Classic-1.9-intel-2016b.eb]: No, I mostly created this to verify that the enhanced easyblock is not broken for DL_Poly Classic without PLUMED, doesn't hurt to include that too.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2316.phanpy.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a4b8b9ed6271fd53f10afe31ac54d8d9 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**